great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -442,6 +433,106 @@ def _build_numeric_columns_rule() -> Rule:
+            column_min_metric_multi_batch_parameter_builder_for_metrics,
+            column_max_metric_multi_batch_parameter_builder_for_metrics,
+            column_median_metric_multi_batch_parameter_builder_for_metrics,
+        ]
+        expectation_configuration_builders: List[ExpectationConfigurationBuilder] = [
+            expect_column_min_to_be_between_expectation_configuration_builder,
+            expect_column_max_to_be_between_expectation_configuration_builder,
+            expect_column_values_to_be_between_expectation_configuration_builder,
+            expect_expect_column_quantile_values_to_be_between_expectation_configuration_builder,
+            expect_column_median_to_be_between_expectation_configuration_builder,
+        ]
+        rule: Rule = Rule(
+            name="numeric_columns_low_precision_rule",
+            variables=variables,
+            domain_builder=numeric_column_type_domain_builder,
+            parameter_builders=parameter_builders,
+            expectation_configuration_builders=expectation_configuration_builders,
+        )
+
+        return rule
+
+    @staticmethod
+    def _build_numeric_columns_high_precision_rule() -> Rule:
+        """
+        This method builds "Rule" object focused on emitting "ExpectationConfiguration" objects for numeric columns.
+        """
+
+        # Step-1: Instantiate "ColumnDomainBuilder" for selecting numeric columns (but not "ID-type" columns).
+
+        numeric_column_type_domain_builder: ColumnDomainBuilder = ColumnDomainBuilder(
+            include_column_names=None,
+            exclude_column_names=None,
+            include_column_name_suffixes=None,
+            exclude_column_name_suffixes=[
+                "_id",
+            ],
+            semantic_type_filter_module_name=None,
+            semantic_type_filter_class_name=None,
+            include_semantic_types=[
+                SemanticDomainTypes.NUMERIC,
+            ],
+            exclude_semantic_types=[
+                SemanticDomainTypes.IDENTIFIER,
+            ],
+            data_context=None,
+        )
+
+        # Step-2: Declare "ParameterBuilder" for every metric of interest.
+
+        column_mean_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.get_column_mean_metric_multi_batch_parameter_builder(
+            json_serialize=True
+        )
+        column_standard_deviation_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.get_column_standard_deviation_metric_multi_batch_parameter_builder(
+            json_serialize=True
+        )
+
+        # Step-3: Declare "ParameterBuilder" for every "validation" need in "ExpectationConfigurationBuilder" objects.
+
+        column_mean_values_range_parameter_builder_for_validations: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.build_numeric_metric_range_multi_batch_parameter_builder(
```

Comment:
@NathanFarmer Is this needed?

Reply:
@alexsherstinsky I just moved it here from the original rule. Looking more closely it is used in the `validation_parameter_builder_configs` of `expect_column_mean_to_be_between_expectation_configuration_builder`. According to the docstring for `DefaultExpectationConfigurationBuilder` this "helps build kwargs needed for this ExpectationConfigurationBuilder". I'm not entirely sure what that means.

Reply:
@alexsherstinsky Yes!